signale.js

```diff
@@ -162,7 +162,7 @@ class Signale {
       if (args[0] instanceof Error) {
         [msg] = args;
       } else {
-        let [{prefix, message, suffix}] = args;
+        const [{prefix, message, suffix}] = args;
         msg = message ? util.format(...message) : util.format(...args);
```

Comment:
Original comment: https://github.com/klauscfhq/signale/pull/23#discussion_r194232847
(just in case the initial notification didn't make it through)